aspnetcore/mvc/views/view-components.md

```diff
@@ -59,6 +59,18 @@ A view component class:
 
 * Doesn't take part in the controller lifecycle, which means you can't use [filters](../controllers/filters.md) in a view component
 
+### The use of NonViewComponent Attribute 
+
+Suppose you have a class in your project with the suffix *ViewComponent*(case-insensitive) and you don't want it to serve as a ViewComponent, you can decorate the class with the attribute `[NonViewComponent]`:
+ 
+```csharp
+     [NonViewComponent]
+     public class ReviewComponent
+     {
+        // .....
+     
+ ```
+
```

Comment:
I think this description is short enough that it doesn't need to be its own section. We also generally avoid "you" where we can, so this suggestion does that and includes a cross-link to the attribute:

```suggestion
To stop a class that has a case-insensitive *ViewComponent* suffix from being treated as a view component, decorate the class with the [[NonViewComponent]](xref:Microsoft.AspNetCore.Mvc.NonViewComponentAttribute) attribute:
 
```csharp
[NonViewComponent]
public class ReviewComponent
{
    // ...
```
```

Reply:
Thank you @serpent5 for the suggestion. I liked the way it was rephrased. it is succinct. I would go ahead and commit the changes.

Reply:
It looks like my suggestion didn't include the closing backtick, so I've just added that in a commit. I'll review the final result and then merge this when it looks correct.

Thanks again, @Maniizzle. ⭐

Reply:
Learnt few things about documentation from this engagement . Thanks @serpent5